appveyor.yml

```diff
@@ -0,0 +1,15 @@
+version: 0.50.9.{build}
+branches:
+  only:
+  - master
+image: Visual Studio 2013
+init:
+- cmd: set PATH=C:\%MSYS2_DIR%\%MSYSTEM%\bin;C:\%MSYS2_DIR%\usr\bin;%PATH%
+environment:
+  COMPILER: msys2
+  BUILDTOOL: cmake
+  MSYS2_DIR: msys64
+  MSYSTEM: MINGW32
+build_script:
+- ps: "mkdir msys2\n\ncd C:/msys64\n\nC:\\msys64\\usr\\bin\\bash -lc (\"for i in {1..3}; do pacman --noconfirm -Syu --force coreutils  && break || sleep 15; done\");\nC:\\msys64\\usr\\bin\\bash -lc \"pacman --noconfirm -Su\"\nC:\\msys64\\usr\\bin\\bash -lc \"pacman --noconfirm -S git\"\n\nC:\\msys64\\usr\\bin\\bash -lc \"pacman --noconfirm -S mingw-w64-i686-gcc\"\n\nC:\\msys64\\usr\\bin\\bash -lc \"pacman --noconfirm -S make\"\n\nC:\\msys64\\usr\\bin\\bash -lc \"pacman --noconfirm -S mingw-w64-i686-cmake\"\nC:\\msys64\\usr\\bin\\bash -lc \"pacman --noconfirm -S mingw-w64-i686-libpng\"\nC:\\msys64\\usr\\bin\\bash -lc \"pacman --noconfirm -S mingw-w64-i686-SDL2\"\nC:\\msys64\\usr\\bin\\bash -lc \"pacman --noconfirm -S mingw-w64-i686-pkg-config\"\n\n#cd C:/projects/ivan/msys2\n\ncd C:/projects/ivan/\n\n#bash -lc \"cd C:/projects/ivan/msys2 && exec 0</dev/null && CXX=g++ CC=gcc cmake -G 'MSYS Makefiles' && make -j2\"\n\ncmake -G \"MSYS Makefiles\" -DCMAKE_CXX_FLAGS=-DWIZARD -DCMAKE_INSTALL_PREFIX=C:/projects/ivan/build \ncmake . \n\nmake -j4 install\n\n#cmake --build . --config Release"
```

Comment:
Can this be split into multiple lines for better readability? E.g. like this: https://github.com/KDE/snorenotify/blob/b8d39da3116f4423e9ae3d9846f4dc0ab3caecf0/appveyor.yml#L4-L20